db/migrate/20230401123456_add_new_fields_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddNewFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    # Add new fields to the users table
+    add_reference :users, :blog, foreign_key: true
+    add_reference :users, :gift_card, foreign_key: true
+    add_reference :users, :user_chanel, foreign_key: true
```

Comment:
The migration file 'AddNewFieldsToUsers' contains a typo in the reference to 'user_chanel', which should be 'user_channel' to match the existing naming conventions. Additionally, adding multiple foreign keys to the users table might not be necessary if the relationships are one-to-many, as this would imply a user can only have one of each. Consider if a join table is more appropriate for any of these relationships.